host/xtest/xtest_4000.c

```diff
@@ -150,6 +151,17 @@ ADBG_CASE_DEFINE(XTEST_TEE_4009, xtest_tee_test_4009,
 		"Description of how to implement ..."
 		 );
 
+ADBG_CASE_DEFINE(XTEST_TEE_4010, xtest_tee_test_4010,
+		/* Title */
+		"Test TEE Internal API create transient object (negative)",
+		/* Short description */
+		"Short description ...",
+		/* Requirement IDs */
+		"TEE-??",
```

Comment:
We should do something about this sooner or later, this stuff is just there, but we never use it.